lib/netconf/netconf.go

```diff
@@ -115,3 +115,61 @@ func Save(network Network, path string) error {
 
 	return nil
 }
+
+type chainJSON struct {
+	ID            uint64 `json:"id"`
+	Name          string `json:"name"`
+	RPCURL        string `json:"rpcurl"`
+	AuthRPCURL    string `json:"auth_rpcurl,omitempty"`
+	PortalAddress string `json:"portal_address"`
+	DeployHeight  uint64 `json:"deploy_height"`
+	IsOmni        bool   `json:"is_omni,omitempty"`
+	BlockPeriod   string `json:"block_period"`
+}
```

Comment:
We don't need this type here. Its only purpose seems to be to serialize/deserialize to/from JSON. Its members match exactly the names of the `Chain` struct. Let's use the `json` tags next to its fields and remove the need for another type.